airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSourceOperations.java

```diff
@@ -370,8 +379,8 @@ public PostgresType getDatabaseFieldType(final JsonNode field) {
     try {
       final String typeName = field.get(INTERNAL_COLUMN_TYPE_NAME).asText().toLowerCase();
       // Postgres boolean is mapped to JDBCType.BIT, but should be BOOLEAN
+      final PostgresType defaultType = PostgresType.valueOf(field.get(INTERNAL_COLUMN_TYPE).asInt(), POSTGRES_TYPE_DICT);
       return switch (typeName) {
```

Comment:
If doable, An array of numerics should also be checked and return an array of integers in case scale is 0.
that's the case of `_numeric`

Reply:
@rodireich got it. Will make the change